samples/drivers/counter/alarm/sample.yaml

```diff
@@ -4,7 +4,7 @@ tests:
   sample.driver.counter.alarm:
     tags: drivers
     harness: console
-    platform_whitelist: nucleo_f746zg
+    platform_whitelist: nucleo_f746zg nrf52840_pca10056 nrf9160_pca10090
```

Comment:
Why not the pca10040, @nordic-krch ?

Reply:
or the pca10028

Reply:
yeah, i can add them as well. wanted to add at least from each series.

Reply:
Yeah, please add the nrf51, if applicable, and perhaps nrf52_pca10040, as this is the "mainstream" 52.

Reply:
done